### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

**Comment by mattfenwick** [comment]
1. nice job digging into this!  This was a seriously tricky issue to come to grips with, but super important for us to solve.  Thanks for taking this on!

2. is the conclusion from this code that we can ship perceptor-scanner in a rhel container, but not in a scratch one?

3. what's the plan for this PR?  I was under the impression that we want to keep upstream perceptor-scanner using a centos image, and that we only want the downstream images (which should be built from a different repo) to use scratch or rhel.  Does that mean that this PR will get moved to the downstream repo?

4. if you do merge this PR, please choose the "squash and merge" option!

**Comment by rrati** [comment]
Not rhel-atomic.  That is a complete mess we don't want to deal with.  Just use base rhel or centos for upstream (probably centos)

File: cmd/perceptor-scanner/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:centos7
+FROM registry.access.redhat.com/rhel7-atomic
```
Comment (rrati) [inline cmd/perceptor-scanner/Dockerfile]: Do not use rhel-atomic.  It's a mess

**Comment by mattfenwick** [comment]
Closing because of the decision to move this feature downstream.

**Pull request closed without merging**